target/linux/ramips/dts/ZL5900V2.dts

```diff
@@ -0,0 +1,99 @@
+/dts-v1/;
+
+#include "rt5350.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "zorlik,zl5900v2", "ralink,rt5350-soc";
+	model = "Zorlik ZL5900V2";
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		lan {
+			label = "zl5900v2:green:lan";
```

Comment:
Wouldn't it make sense to use this LED for the lan port status? Add a section for your board to `target/linux/ramips/base-files/etc/board.d/01_leds` with

```
zorlik,zl5900v2)
	ucidef_set_led_netdev "lan" "lan" "$boardname:green:lan" eth0
	;;
```

Reply:
Done.